src/chain/bitcoind.rs

```diff
@@ -329,9 +329,7 @@ impl BitcoindChainSource {
 				},
 				Err(e) => {
 					log_error!(self.logger, "Failed to poll for chain data: {:?}", e);
-					let res = Err(Error::TxSyncFailed);
-					self.wallet_polling_status.lock().unwrap().propagate_result_to_subscribers(res);
```

Comment:
The cleanup here, does that mean that the previous commit would send redundant updates?